### ISSUE TITLE
Continuation of Flicker Discussion

### ISSUE THREAD

**Issue by Zieg777** [Mithion/ArsMagica2#1403]
I know there were people talking about flickers on the "Mage Bag" Issue page. This is to continue that discussion if anyone wants.

I'd like to start by asking Mithion a question. You mentioned that Flickers' range can be upgraded to be "quite large" I'd love to be able to make my item transport network cover more area. What upgrade is supposed to do this? It's not listed in the compendium.

**Comment by ThatOneTraitor** [comment]
He said he was planning on making it so it can be upgraded to be quite large. it isn't in yet, it was just an idea.

**Comment by jjtParadox** [comment]
Anyone have interesting ideas as to how that upgrade should be done or crafted?

**Comment by ChilledDayz** [comment]
I think Mithion said that they can already be upgraded

**Comment by ThatOneTraitor** [comment]
Maybe make a super essence by adding a bse essence + 8 of a type of essence, different types = different upgrades

**Comment by Mithion** [comment]
You mean like a pure essence?

I think it should remain consistent with the other forms of upgrading flicker habitats - adjacent habitats with different flicker types in them.  The flicker type determines the nature of the upgrade.

**Comment by ChilledDayz** [comment]
So maybe just an air flicker into a habitat?

**Comment by Mithion** [comment]
Something like that, yeah.  Could also only apply the range mod if it has power.

**Comment by ChilledDayz** [comment]
Yeah I think power would be a must as I feel like the flickers would need power to make a larger jump. :-)

**Comment by ChilledDayz** [comment]
Maybe make it neutral essence, so using flicker over distance is more expensive.

**Comment by Zieg777** [comment]
Honestly, Neutral is the cheapest power. Calefactors make it the easiest to get. It's much faster generating than light by far.

Regardless though, Don't all the flicker upgrades require power? Namely speed/lightning flickers? I generally give em power anyway if they're within range of existing systems.

**Comment by ThatOneTraitor** [comment]
i meant as in make a recipe with 1 base essence core in the middle and 8 essences of a certain type, and it would give a certain effect, like maybe 8 ender essence + 1 base essence gives the flicker the ability to teleport to another flicker habitat.

**Comment by Zieg777** [comment]
Perhaps add essences to a flicker in a jar to add traits? 8 Ender essence around an air flicker jar. Put that new modified flicker into the adjacent habitat to allow the flickers to teleport?

**Comment by ThatOneTraitor** [comment]
what @Zieg777 said is what i was trying to say
